src/components/FormStepSummary/ComponentValueDisplay.js

```diff
@@ -209,6 +209,16 @@ const CoSignDisplay = ({component, value}) => {
   return <CoSignOld interactive={false} />;
 };
 
+const AddressNLDisplay = ({component, value}) => {
+  if (!value) {
+    return <EmptyDisplay />;
+  }
+
+  return `${value.postcode} ${value.houseNumber}${value.houseLetter || ''} ${
```

Comment:
adding a story will take care of the coverage & protect us against visual regressions in the future